README.md

```diff
@@ -2,6 +2,68 @@
 
 A project that presents an iRODS Zone as HTTP v1.1.
 
+## Quickstart (Running from Docker Hub Image)
+
+Generate a local configuration JSON file.
+
+```
+docker run --rm irods/irods_http_api --dump-config-template > config.json
+```
+
+Edit/update the template `config.json` file (point to your iRODS Zone, remove OIDC and TLS sections, etc.).
+
+```
+vim config.json
+```
+
+Launch the HTTP API with your customized configuration file to check for success/errors:
+
+```
+docker run --rm --name irods_http_api \
+    -v ./config.json:/config.json:ro \
+    -p 9000:9000 \
+    irods/irods_http_api
+```
+
+Then, the HTTP API will be available:
+```
+$ curl -X POST -u rods:rods \
+ http://localhost:9000/irods-http-api/0.1.0/authenticate
+568bbfc2-7d19-4723-b659-bb9325f9b076
+
+$ curl -s http://localhost:9000/irods-http-api/0.1.0/collections \
+    -H 'Authorization: Bearer 568bbfc2-7d19-4723-b659-bb9325f9b076' \
+    --data-urlencode 'op=stat' \
+    --data-urlencode 'lpath=/tempZone/home/rods' -G | jq
+{
+  "inheritance_enabled": false,
+  "irods_response": {
+    "status_code": 0
+  },
+  "modified_at": 1699448576,
+  "permissions": [
+    {
+      "name": "rods",
+      "perm": "own",
+      "type": "rodsadmin",
+      "zone": "tempZone"
+    },
+    {
+      "name": "alice",
+      "perm": "read_object",
+      "type": "groupadmin",
+      "zone": "tempZone"
+    }
+  ],
+  "registered": true,
+  "type": "collection"
+}
+```
```

Comment:
This part feels like it should be part of a slide deck rather than a quickstart because it shows the commands along with their output.

It feels like this should be the commands followed by a section saying "If everything succeeded, you should see output similar to the following."

For example:
```
bearer_token=$(curl -X POST ...)
curl -G -H "Authorization: Bearer $bearer_token" ...
```

Thoughts?

Reply:
Well, I want the first-time reader to 'see' the output and get what success looks like in the first couple glances.

Hiding the token is cleaner, but ... doesn't feel as instructive.

Happy to tweak formatting to convey 'command' and then 'output' a little better.  But I think this is the right balance of 'showing' rather than 'telling'.